files/uk/web/html/element/input/time/index.md

```diff
@@ -9,40 +9,10 @@ browser-compat: html.elements.input.type_time
 
 Елементи {{HTMLElement("input")}} типу **`time`** (час) утворюють поля введення, призначені для того, щоб дозволити користувачам легко вводити час (години та хвилини, а також, за бажанням, секунди).
 
-Користувацький інтерфейс цього елемента керування відрізняється в різних браузерах; дивіться докладніше про це в розділі [Сумісність із браузерами](#sumisnist-iz-brauzeramy). У браузерах, що його не підтримують, цей елемент доладно відступає до [`<input type="text">`](/uk/docs/Web/HTML/Element/input/text).
+Попри те, що зовнішній вигляд користувацького інтерфейсу цього контрольного елемента залежить від браузера та операційної системи, можливості в нього одні й ті ж. Значення завжди є 24-годинним відформатованим часом `hh:mm` або `hh:mm:ss`, з додатковими нулями, незалежно від формату введення UI.
```

Comment:
```suggestion
Попри те, що зовнішній вигляд користувацького інтерфейсу цього контрольного елемента залежить від браузера та операційної системи, можливості в нього одні й ті ж. Значення завжди є 24-годинним відформатованим часом `hh:mm` або `hh:mm:ss` з провідними нулями, незалежно від формату введення UI.
```
https://uk.wikipedia.org/wiki/%D0%9F%D1%80%D0%BE%D0%B2%D1%96%D0%B4%D0%BD%D1%96_%D0%BD%D1%83%D0%BB%D1%96